### PR TITLE
[W5.4][W14-A2]Ritchie Lam Qiaowei

### PR DESCRIPTION
**Pull request by ritchielq**
Added ChangeTag command

### CODE REVIEW

**Comment by LeonChowWenHao** [comment]
Followed coding & OOP style. Suitable tests created to test the newly added command.

Suggest to spread out changes to smaller commits in case of unexpected errors.

File: test/java/seedu/addressbook/commands/ChangeTagCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+    /**
+     * Asserts that the person at the specified index can be successfully modified.
+     *
+     * The addressBook passed in will not be modified (no side effects).
+     *
+     * @throws PersonNotFoundException if the selected person is not in the address book
+     */
+    private void assertChangeTagSuccessful(int targetVisibleIndex, AddressBook addressBook,
+                                          List<ReadOnlyPerson> displayList, Set<String> newTags) throws PersonNotFoundException, IllegalValueException {
```
Comment (okkhoy) [inline test/java/seedu/addressbook/commands/ChangeTagCommandTest.java]: this is a particularly long line. consider splitting

**Comment by okkhoy** [comment]
@ritchielq 
Some comments added. Please ack & close the PR after reading comments.

**Comment by ritchielq** [comment]
Noted. Will try to be more careful with the coding standards.

**Pull request closed without merging**